lighthouse-core/scripts/roll-to-devtools.sh

```diff
@@ -21,7 +21,7 @@ else
   frontend_dir="$chromium_dir/third_party/blink/renderer/devtools/front_end"
 fi
 
-tests_dir="$frontend_dir/../../../web_tests/http/tests/devtools/audits2"
+tests_dir="$frontend_dir/../../../web_tests/http/tests/devtools/audits"
```

Comment:
totally slipped on this

Reply:
this only affects the test fixture/expectation updates though right?